timeless/views.py

```diff
@@ -17,6 +13,11 @@ class UserView(AdminView):
 @todo #173:30min Once DeleteView is implemented, refactor all blueprint views
  to use it for validating the form and deleting the record in the database.
  Reuse SingleObjectMixin to provide simple solution to fetch by id.
+@todo #309:30min Refactor all blueprint views to use ListView for getting the
+ list of objects from db using model. Also, make sure list.html template is
+ made generic to allow all other views to use it. Feel free to add more puzzles
+ since there are a lot of views. Already refactored views: TableListView. Check
+ in issues to see if some view is not being refactored in other issue.
```

Comment:
@paulodamaso Can you enhance this puzzle a bit? Specify that it's already been done for Tables, so they have an example.

Reply:
@amihaiemil `Already refactored views: TableListView` is not enough?

Reply:
@paulodamaso missed that, sorry :D